apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/OpenOrders.tsx

```diff
@@ -230,24 +223,29 @@ export const OpenOrders = () => {
     const totalReturned = `${fromTokenAmount} ${fromTokenInfo?.ticker}`
     const collateralUtxo = await getCollateralUtxo()
 
-    const fee = await getFee(utxo, collateralUtxo, bech32Address)
-
-    openModal(
-      strings.listOrdersSheetTitle,
-      <ModalContent
-        assetFromIcon={<TokenIcon wallet={wallet} tokenId={fromTokenInfo?.id ?? ''} variant="swap" />}
-        assetToIcon={<TokenIcon wallet={wallet} tokenId={toTokenInfo?.id ?? ''} variant="swap" />}
-        onConfirm={() => onOrderCancelConfirm(order)}
-        onBack={closeModal}
-        assetFromLabel={assetFromLabel}
-        assetToLabel={assetToLabel}
-        assetAmount={`${tokenAmount} ${assetToLabel}`}
-        assetPrice={`${tokenPrice} ${assetFromLabel}`}
-        totalReturned={totalReturned}
-        fee={fee}
-      />,
-      460,
-    )
+    try {
+      const fee = await getFee(utxo, collateralUtxo, bech32Address)
+      openModal(
+        strings.listOrdersSheetTitle,
+        <ModalContent
+          assetFromIcon={<TokenIcon wallet={wallet} tokenId={fromTokenInfo?.id ?? ''} variant="swap" />}
+          assetToIcon={<TokenIcon wallet={wallet} tokenId={toTokenInfo?.id ?? ''} variant="swap" />}
+          onConfirm={() => onOrderCancelConfirm(order)}
+          onBack={closeModal}
+          assetFromLabel={assetFromLabel}
+          assetToLabel={assetToLabel}
+          assetAmount={`${tokenAmount} ${assetToLabel}`}
+          assetPrice={`${tokenPrice} ${assetFromLabel}`}
+          totalReturned={totalReturned}
+          fee={fee}
+        />,
+        460,
+      )
+    } catch (error) {
+      if (error instanceof Error) {
```

Comment:
is this if required?

Reply:
in that case we need also an alert on the else side, if somewhere deep something else is thrown.